dt/Adoption/DFG-upgrades-adoption-18.0-from-17.1-passed-phase2-rhev-3cont-bm-2comp-ipv4-HA-geneve-ovn-bm-3ocp/DFG-upgrades-adoption-18.0-from-17.1-passed-phase2-rhev-3cont-bm-2comp-ipv4-HA-geneve-ovn-bm-3ocp.md

```diff
@@ -0,0 +1,48 @@
+# Deployed Topology DFG-upgrades-adoption-18.0-from-17.1-passed-phase2-rhev-3cont-bm-2comp-ipv4-HA-geneve-ovn-bm-3ocp
+
+## General information
+
+| Revision | Change                | Date             |
+|--------: | :-------------------- | :--------------: |
+| v0.1     | Initial publication   | 22.12.23      |
+
+## Node topology
+| Node role                                     | bm/vm | amount |
+| --------------------------------------------- | ----- | ------ |
+| Openshift master/worker combo-node cluster    | bm    | 3      |
```

Comment:
IMO, we can use a bm to deploy a virtual OCP control plane. In this way, we can avoid the other topology as the primary goal is RHV + OCP